app/controllers/api/lists_controller.rb

```diff
@@ -0,0 +1,20 @@
+class Api::ListsController < ApiController
+  before_action :authenticate_user
+
+  def create
+    user = User.find(params[:user_id])
```

Comment:
Use the `@current_user` idea from above otherwise this code means I can create a list for anyone in the database provided I have their ID.